Library/Homebrew/sorbet/tapioca/compilers/args.rb

```diff
@@ -0,0 +1,74 @@
+# typed: strict
+# frozen_string_literal: true
+
+require_relative "../../../global"
+
+module Tapioca
+  module Compilers
+    class Args < Tapioca::Dsl::Compiler
+      # This is ugly, but we're moving to a new interface that will use a consistent DSL
+      # These are cmd/dev-cmd methods that end in `_args` but are not parsers
+      NON_PARSER_ARGS_METHODS = T.let([
+        :formulae_all_installs_from_args,
+        :reproducible_gnutar_args,
+        :tar_args,
+      ].freeze, T::Array[Symbol])
+
+      # FIXME: Enable cop again when https://github.com/sorbet/sorbet/issues/3532 is fixed.
+      # rubocop:disable Style/MutableConstant
+      ConstantType = type_member { { fixed: T.class_of(Homebrew::CLI::Args) } }
+      # rubocop:enable Style/MutableConstant
```

Comment:
I don't follow where you use this -- care to elaborate?

Reply:
Per Tapioca [docs](https://github.com/Shopify/tapioca?tab=readme-ov-file#writing-custom-dsl-compilers):

> Every compiler must declare the type member ConstantType in order for Sorbet to understand what the return type of the constant attribute reader is. It needs to be assigned the correct type variable matching the type of constants that gather_constants returns. This generic variable allows Sorbet to type-check method calls on the constant reader in your decorate method. See the Sorbet documentation on [generics](https://sorbet.org/docs/generics) for more information.